appeals/web/src/server/lib/page-components/text.js

```diff
@@ -0,0 +1,66 @@
+import { mapActionComponent } from '#lib/mappers/component-permissions.mapper.js';
+import { permissionNames } from '#environment/permissions.js';
+import { SHOW_MORE_MAXIMUM_CHARACTERS_BEFORE_HIDING } from '#lib/constants.js';
+
+/**
+ * Returns the instructions for a text display field
+ *
+ * @param {Object} options
+ * @param {string} options.id
+ * @param {string} options.text
+ * @param {string|{html: string}|null} [options.value]
```

Comment:
You could potentially use the type `HtmlProperty` defined in `appeals/web/src/server/lib/nunjucks-template-builders/gds-components-properties-jsdoc.js` here, but that does also include an optional `pageComponents` property - just a thought